### ISSUE TITLE
ArsMagica2 Incompatability with Forge Essentials 1.10.2 causes Crash

### ISSUE THREAD

**Issue by MapleSight** [Growlith1223/ArsMagica2#290]
See [Here](https://github.com/ForgeEssentials/ForgeEssentials/issues/2233)

Crash https://hastebin.com/omasoqijit.sql

It would be very very nice if you could look into why there's an incompatibility issue. Thank you for your time.

[Also posted it here.](https://github.com/Mithion/ArsMagica2/issues/1605)

**Comment by Growlith1223** [comment]
well, the crash does say to blame dinnerbone, so blame dinnerbone!

All joking aside, i'll check it out, not sure why it's crashing but regardless, i'll check it out!

**Comment by MapleSight** [comment]
Thank you, keep me updated on whatever progress you make.

Comparing 1.7.10, There were issues in 1.7.10 but the mods didn't insta-crash the server like it does now. Getting it back to the  crash every now and then would be really, really appreciated.

**Comment by MapleSight** [comment]
@Growlith1223 Heya, was wondering if you had time to look into this issue yet.

**Comment by D-Fink** [comment]
Yeah, experiencing same issue. Its telling me that am2 fire guardian is crashing it.

**Comment by TheIcyOne** [comment]
@abloodyfetus  do you have a crash report for that?

**Comment by D-Fink** [comment]
Sorry I dunno how to do that but it appears to be same as the first one posted at the top. exception from am2 fire guardian. first line :p

**Comment by D-Fink** [comment]
https://hastebin.com/ovozupigec.sql - CRASH

**Comment by Scratchman** [comment]
I've just installed forgeessentials-1.10.2-10.1.109-server on a test server and we're having the same crash, the server was fine with ArsMagica2-1.10.2-1.5.0-18 before forge essentials was added.

[crash report](https://pastebin.com/MndbCJ2T)

**Comment by TheIcyOne** [comment]
https://github.com/Mithion/ArsMagica2/issues/1562

Relevant issue on the original repo.

**Comment by Chrisdoyle11** [comment]
Hi,

I'm experiencing the same issue here. I'm using the "All The Mods" mod pack that comes with Ars Magica 2 included. The mod pack runs well out of the box. Installing Forge Essentials (forgeessentials-1.10.2-10.2.1374-server.jar) causes the below error. I require Forge Essentials as an admin tool for my server.

[https://pastebin.com/0GQtu7yd](https://pastebin.com/0GQtu7yd)

**Comment by eleksploded** [comment]
Hey,
I'm getting the same problem: [https://pastebin.com/A5Pxxq5L](https://pastebin.com/A5Pxxq5L)
I would like to see this fixed.
